products/compute/api.yaml

```diff
@@ -10528,6 +10530,36 @@ objects:
                   - snapshot_schedule_policy.0.snapshot_properties.0.guest_flush
                 description: |
                   Whether to perform a 'guest aware' snapshot.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'groupPlacementPolicy'
+        conflicts:
+          - 'snapshotSchedulePolicy'
+        description: |
+          Policy for creating snapshots of persistent disks.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'vmCount'
+            at_least_one_of:
+              - group_placement_policy.0.vm_count
+              - group_placement_policy.0.availability_domain_count
+            description: |
+              Number of vms in this placement group
+          - !ruby/object:Api::Type::Integer
+            name: 'availabilityDomainCount'
+            at_least_one_of:
+              - group_placement_policy.0.vm_count
+              - group_placement_policy.0.availability_domain_count
+            description: |
+              The number of availability domains instances will be spread across. If two instances are in different
+              availability domain, they will not be put in the same low latency network
+          - !ruby/object:Api::Type::Enum
+            name: 'collocation'
+            description: |
+              Collocation specifies whether to place VMs inside the same availability domain on the same low-latency network.
+              Specify `COLLOCATED` to enable collocation. Can only be specified with `vm_count`.
+            values:
+              - :UNSPECIFIED_COLLOCATION
```

Comment:
```suggestion
```

We can probably omit that one? In proto, UNSPECIFIED_X is the equivalent of an error so APIs shouldn't use it.